src/main/scala/io/github/gaelrenoux/tranzactio/doobie/package.scala

```diff
@@ -1,19 +1,18 @@
 package io.github.gaelrenoux.tranzactio
 
-import java.sql.{Connection => JdbcConnection}
-
 import _root_.doobie.free.KleisliInterpreter
 import _root_.doobie.util.transactor.{Strategy, Transactor}
 import cats.effect.Resource
 import io.github.gaelrenoux.tranzactio.test.DatabaseModuleTestOps
-import io.github.gaelrenoux.tranzactio.utils.ZCatsBlocker
 import izumi.reflect.Tag
 import zio.blocking.Blocking
 import zio.interop.catz._
+import zio.interop.catz.implicits.rts
```

Comment:
I would avoid importing this implicit inside library. It will cause, that default `Blocker` and `Clock` will be used. This can cause various problems. E.g. I use https://kamon.io/ so I have to provide my own ExecutionContexts (also for `Blocking`) and with this import the instrumentation does not work. I think in the library it would be better to require `Blocking` and `Clock` to be provided and then use it as in this example: https://github.com/zio/interop-cats#async (see note https://github.com/zio/interop-cats#easier-imports-at-a-cost)

Reply:
@hmemcpy I created related MR in your repository: https://github.com/hmemcpy/tranzactio/pull/1